shared/src/vyxal/MiscHelpers.scala

```diff
@@ -199,4 +203,18 @@ object MiscHelpers:
       case (a: VList, b: VList) => VList.from((a, b).zipped.map(subtract(_, _)))
       case (a, b) => ???
 
+  def untilNoChange(function: VFun, value: VAny)(using Context): VAny =
```

Comment:
```suggestion
  def untilNoChange(function: VFun, value: VAny)(using Context): VList =
```
Just making it more specific (unless making it a `VList` messes things up elsewhere, in which case, ignore this)